src/planwise/client/population.cljs

```diff
@@ -0,0 +1,48 @@
+(ns planwise.client.population
+  (:require [re-frame.core :as rf]
+            [re-frame.core :refer [dispatch subscribe]]
+            [planwise.client.ui.rmwc :as m]))
+;; ----------------------------------------------------------------------------
+;; API methods
+
+(def load-population-sources
+  {:method    :get
+   :section   :show
+   :uri       "/api/population"})
+
+;; ----------------------------------------------------------------------------
+;; Listing population sources
+
+(rf/reg-event-fx
+ :population/load-population-sources
+ (fn [{:keys [db]} [_]]
+    {:api (assoc load-population-sources
+              :on-success [:population/sources-loaded])}))
+
+(rf/reg-event-db
+ :population/sources-loaded
+ (fn [db [_ population-sources]]
+   (assoc-in db [:population :list] population-sources)))
+
+;; ----------------------------------------------------------------------------
+;; Subs
+
+(rf/reg-sub
+ :population/list
```

Comment:
let's call it `:population/dropdown-options` instead of `:population/list`. That way it is similar to how the dataset2 subs are.